js/navbar.js

```diff
@@ -1,3 +1,17 @@
+window.addEventListener('DOMContentLoaded',(event)=>{
+  const checkURL = window.location.href;
+  const stagingURL = "https://staging-welcome.realdevsquad.com/";
+  const productionURL = "https://welcome.realdevsquad.com/";
+
+  if (checkURL == stagingURL) {
```

Comment:
Let's check for production URL here, by that we will have stagingLoginURL for all other URLs.

Reply:
Will change it.